inst/htmlwidgets/plugins/sortable-rstudio/rank_list.css

```diff
@@ -4,7 +4,6 @@
   border: 1px solid #ddd;
   padding: 10px;
   margin: 5px;
-  display: flex;
```

Comment:
this one is needed to have the horizontal containers wrap to the bottom on a skinny window, correct?